Middleware/SVGElements/Text.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+public class Text
+{
+    public Text() 
+    {
+
+    }
+
+    public async Task<string> SVG(string wrapper, IWebHostEnvironment _env, string textX, string textY, string fontWeight, string fontSize, string textColor, string fill, string textDecoration, string opacity, string text, string animations) 
+    {
+      string svg = await _env.ReadFileFromWebRootAsync("text.svg");
```

Comment:
Does text.svg have to be stored in the web root? Is this the only place it's used? If so, I'd be tempted to inline the string to C# to avoid making every caller of this have to be async. Unless it's a huge file.